docs/course_materials/exercises/check_helper.py

```diff
@@ -50,12 +75,12 @@ def created_all_paths(exercise_paths):
     num_of_created_paths = 0
     print("Checking created directories or files...")
     for cur_path in exercise_paths:
-        prYellow("'" + cur_path + "'")
+        print_yellow("'" + HOME + cur_path + "'")
         print(" ... ", end='')
-        if not os.path.exists(cur_path):
-            prRed("Missing")
+        if not os.path.exists(HOME + cur_path):
```

Comment:
created_all_paths and deleted_all_paths content is pretty much the same, can you extract the content to a function?